docs/manipulate/add.md

```diff
@@ -6,7 +6,7 @@ title: Add
 Returns a cloned Day.js object with a specified amount of time added.
 
 ```js
-dayjs().add(7, 'day')
+result = dayjs().add(7, 'day')
```

Comment:
const a = dayjs
const b = dayjs().add xxx

// a -> the original value and will not change
// b -> the manipulation result


is it better?

Reply:
Like this:
const a = dayjs
const b = a.add xxx

this would be also helpfull.